pt-BR/mail/chrome/messenger/am-server-top.dtd

```diff
@@ -2,12 +2,12 @@
 
 <!ENTITY serverSettings.label "Receber mensagens">
 <!-- LOCALIZATION NOTE (popServer.label) : do not translate "POP" in below line -->
-<!ENTITY popServer.label "POP Mail Server">
+<!ENTITY popServer.label "Servidor de emails POP">
 <!-- LOCALIZATION NOTE (imapServer.label) : do not translate "IMAP" in below line -->
-<!ENTITY imapServer.label "IMAP Mail Server">
+<!ENTITY imapServer.label "Servidor de emails IMAP">
```

Comment:
@brunotiussi I understand your position, however my recommendation is to translate all possible words to PT-BR. If you'd like to keep it short (GUI purposes) either use e-mail or email (chose one to keep consistency). Consider using singular.